src/rfq/protocols/hashflow/client.rs

```diff
@@ -433,35 +431,68 @@ impl RFQClient for HashflowClient {
 
                     let mut quote_attributes: HashMap<String, Bytes> = HashMap::new();
                     quote_attributes.insert("pool".to_string(), quote.quote_data.pool);
+                    if let Some(external_account) = quote.quote_data.external_account {
+                        quote_attributes.insert("external_account".to_string(), external_account);
+                    }
                     quote_attributes.insert("trader".to_string(), quote.quote_data.trader);
+                    if let Some(effective_trader) = quote.quote_data.effective_trader {
+                        quote_attributes.insert("effective_trader".to_string(), effective_trader);
+                    }
```

Comment:
I thought we didn't need this in the end..

Reply:
but for our case it will always be the zero address correct? we can save a tiny bit of gas if we don't pass this and just define it in the executor